lib/node_modules/@stdlib/math/special/data/unary_function_database.json

```diff
@@ -0,0 +1,3386 @@
+{
+  "abs": {
+    "input_dtypes": "numeric_and_generic",
+    "output_dtypes": "real_and_generic",
+    "excluded_dtypes": [
+      "float16",
+      "int64",
+      "uint64",
+      "uint8c",
+      "complex32"
+    ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/labs",
+      "int16": "@stdlib/math/base/special/labs",
+      "int32": "@stdlib/math/base/special/labs",
+      "uint8": "@stdlib/number/uint8/base/identity",
+      "uint16": "@stdlib/number/uint16/base/identity",
+      "uint32": "@stdlib/number/uint32/base/identity",
+      "float32": "@stdlib/math/base/special/absf",
+      "float64": "@stdlib/math/base/special/abs",
+      "complex64": "@stdlib/math/base/special/cabsf",
+      "complex128": "@stdlib/math/base/special/cabs",
+      "generic": "@stdlib/math/base/special/abs"
+    },
+    "policies": {
+      "output": "real_and_generic",
+      "casting": "none"
+    }
+  },
+  "abs2": {
+    "input_dtypes": "numeric_and_generic",
+    "output_dtypes": "real_and_generic",
+    "excluded_dtypes": [
+      "float16",
+      "int64",
+      "uint64",
+      "uint8c",
+      "complex32"
+    ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/abs2f",
+      "int16": "@stdlib/math/base/special/abs2f",
+      "int32": "@stdlib/math/base/special/abs2f",
+      "uint8": "@stdlib/math/base/special/abs2f",
+      "uint16": "@stdlib/math/base/special/abs2f",
+      "uint32": "@stdlib/math/base/special/abs2f",
```

Comment:
```suggestion
```

We should remove these, as `abs2f` is not universally appropriate. It depends on the output dtype what kernel should be used. That applies here and for other functions in this file.